README.md

```diff
@@ -57,62 +69,45 @@ py -3 unrpyc.py [options] script1 script2 ...
-                        The generated code is exactly equivalent, only slightly more cluttered.
-  --try-harder          Tries some workarounds against common obfuscation
-                        methods. This is a lot slower.
+  -h, --help            Show this help message and exit
+  -c, --clobber         Overwrites existing output files
+  --try-harder          Tries some workarounds against common obfuscation methods. This is a
+                        lot slower.
+  -p, --processes       Use the specified number or processes to decompile. Defaults to the
+                        amount of hw threads available minus one, disabled when muliprocess-
+                        ing is unavailable.
+  -t TRANSLATE,
+  --translate TRANSLATE Switches the apps main language to the the given language. This
+                        string must be present in the tl dir and a exact match.
+  -d, --dump            Instead of decompiling, pretty print the ast to a file.
+  --comparable          Only for dumping, remove several false differences when comparing
+                        dumps. This suppresses attributes that are different even when the
+                        code is identical, such as file modification times.
+  --no-pyexpr           Only for dumping, disable special handling of PyExpr objects, instead
+                        printing them as strings. This is useful when comparing dumps from
+                        different versions of Ren'Py. It should only be used if necessary,
+                        since it will cause loss of information such as line numbers.
+  --no-init-offset      By default, unrpyc attempt to guess when init offset statements were
+                        used and insert them. This is always safe to do for Ren'Py 8, but as
+                        it is based on a heuristic it can be disabled. The generated code is
+                        exactly equivalent, only slightly more cluttered.
   --register-sl-displayable SL_CUSTOM_NAMES [SL_CUSTOM_NAMES ...]
-                        Accepts mapping separated by '=', where the first
-                        argument is the name of the user-defined displayable
-                        object, and the second argument is a string containing
-                        the name of the displayable,potentially followed by a
-                        '-', and the amount of children the displayable
-                        takes(valid options are '0', '1' or 'many', with
-                        'many' being the default)
+                        Accepts mapping separated by '=', where the first argument is the na-
+                        me of the user-defined displayable object, and the second argument is
+                        a string containing the name of the displayable,potentially followed
+                        by a '-', and the amount of children the displayable takes(valid op-
+                        tions are '0', '1' or 'many', with 'many' being the default)
```

Comment:
This entire block is autogenerated, I'd prefer just changing this before release once we're done with the actual work. Otherwise I'll have to make my comments twice.

Reply:
OK